$NetBSD: patch-configure,v 1.1 2023/01/16 01:24:36 dholland Exp $

If we find Solaris, set the define the way the source actually expects
it to be set, instead of provoking a cpp syntax error.

--- configure~	2022-03-02 11:49:09.000000000 +0000
+++ configure
@@ -2299,7 +2299,7 @@ case $host_os in *\ *) host_os=`echo "$h
 case "$host" in
 	*sun5* | *solaris2*)
 
-$as_echo "#define SOLARIS /**/" >>confdefs.h
+$as_echo "#define SOLARIS 1 /**/" >>confdefs.h
 
 	;;
 esac
